src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -345,4 +356,12 @@ public ArticlePageObject clickPublishButton() {
     submitButton.click();
     return new ArticlePageObject();
   }
+
+  public void waitForDraftToBeSaved() {
+    try {
+      Thread.sleep(5001);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)